internal/cmd/kafka/confluent_kafka.go

```diff
@@ -245,6 +246,9 @@ func RunConsumer(consumer *ckafka.Consumer, groupHandler *GroupHandler) error {
 		select {
 		case <-signals: // Trap SIGINT to trigger a shutdown.
 			output.ErrPrintln(errors.StoppingConsumerMsg)
+			if _, err := consumer.Commit(); err != nil {
+				log.CliLogger.Warnf("Failed to commit current consumer offset: %v", err)
+			}
 			consumer.Close()
```

Comment:
@brianstrauch defer wouldn't work as consumer has been closed and connection is gone. so i'm adding a `commit` with timeout on line 262, as `commit` would hang if there's no connection.